index.js

```diff
@@ -1,11 +1,29 @@
-function scuberGreetingForFeet(){
-  // Write your code here!
+function scuberGreetingForFeet(ride){
```

Comment:
Is there a more descriptive name for this argument than "ride"? Is what's being passed in really a ride? What else could this be called that would be more accurate?
```suggestion
function scuberGreetingForFeet(ride){
```